pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -1593,7 +1593,8 @@ async def run(self):
                             if incorrect_modular_relation:
                                 # Covers #2643
                                 result.is_modular = True
-
+                            if len(result.files) != len(d_content.content.files):
```

Comment:
Could you break this out into a separate var like the others?  It helps when debugging